designs/panorama_plugin/panorama_plugin.md

```diff
@@ -0,0 +1,65 @@
+# Panorama Security Plugin
+This is a security appliance plugin for the Panorama Security Management Appliance, which orchestrates PaloAlto virtual firewalls deployed by OSC. The plugin shall conform to the [security manager api](https://github.com/opensecuritycontroller/security-mgr-api) specification. We propose to have:
+- A Panorama Dynamic address group for each OSC security policy. It will be labeled by a tag and targeted by a _Panorama_ security policy.
```

Comment:
not sure i understand this point. remove? 

The DAG on PAN does not need to be labelled by a tag. The DAG will use the shared tags for their match criteria